R/admiraldev-package.R

```diff
@@ -2,8 +2,7 @@
 #' @importFrom dplyr across arrange bind_rows case_when desc ends_with filter
 #'   full_join group_by if_else mutate n pull rename row_number select slice
 #'   starts_with transmute ungroup n_distinct union distinct summarise coalesce
-#'   bind_cols na_if tibble
-#' @importFrom magrittr %>%
+#'   bind_cols na_if tibble %>%
```

Comment:
By the way, it seems that `bind_cols`, `na_if`, and `tibble` are not used in admiraldev.

Is there an easy way to check which of the imported functions are actually used?

Reply:
This will be resolved in issue https://github.com/pharmaverse/admiraldev/issues/435